### PR TITLE
Fix misleading definition of CORS

### PR DESCRIPTION
**Pull request by remyrd**

### CODE REVIEW

**Comment by pfrazee** [comment]
Hmm I see what you're doing but maybe we can do better.

"CORS restrictions are not applied"

**Comment by remyrd** [comment]
> Hmm I see what you're doing but maybe we can do better.
> 
> "CORS restrictions are not applied"

I may be wrong but SOP is the default restriction applied to scripts in most browsers. CORS is the more liberal "query whatever you want" policy. So I would rephrase "SOP restrictions are not applied"

**Comment by pfrazee** [comment]
Yeah but we tend to think specifically of CORS in this case because that's the mechanism by which SOP is enforced.

**Comment by remyrd** [comment]
You are right. Apologies for the misconception!
